front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -163,14 +165,18 @@ const ManchetteWithSpaceTimeChartWrapper = ({
   return (
     <div className="manchette-space-time-chart-wrapper">
       <div className="header">
-        <ManchetteMenuButton setWaypointsPanelIsOpen={setWaypointsPanelIsOpen} />
-        {waypointsPanelIsOpen && waypointsPanelData && (
-          <WaypointsPanel
-            waypointsPanelIsOpen={waypointsPanelIsOpen}
-            setWaypointsPanelIsOpen={setWaypointsPanelIsOpen}
-            waypoints={operationalPoints}
-            waypointsPanelData={waypointsPanelData}
-          />
+        {allowWayPointsModification && (
```

Comment:
Could we maybe only display this menu if `waypointsPanelData` is not undefined? It seems like it's only set for operational studies and is required to populate the menu.